ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1393 @@
+{
```

Comment:
<p>All the import should be in the beginning</p><p> </p><p>Remove commented lines</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>